### PR TITLE
Adding CLIFramework

### PR DESCRIPTION
**Pull request by c9s**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -446,7 +446,7 @@ Libraries to help manage database schemas and migrations.
 * [Phinx](https://github.com/robmorgan/phinx) - Another database migration library.
 * [Migrations](https://github.com/icomefromthenet/Migrations) - A migration management library.
 * [Doctrine Migrations](http://docs.doctrine-project.org/projects/doctrine-migrations/en/latest/toc.html) - A migration library for Doctrine.
-
+C
```
Comment (tleb) [inline README.md]: Why is this ?

Comment (c9s) [inline README.md]: ah I think its a typo

**Comment by c9s** [comment]
The typo is removed.

**Comment by c9s** [comment]
Hi @jadb This PR is also updated, please take a look, thank you

**Comment by jadb** [comment]
Not sure I want it to be prepended to the existing list. Please move to the bottom of this category's list (they aren't alphabetical or by popularity).

**Comment by c9s** [comment]
Hi @jadb, the item is now at the end of the list. Thanks

**Comment by jadb** [comment]
Thanks :)

**Pull request merged**